pkgs/top-level/all-packages.nix

```
@@ -1678,6 +1678,11 @@ with pkgs;
 
   etlegacy = callPackage ../games/etlegacy { lua = lua5_4; };
 
+  fastfetch = darwin.apple_sdk_11_0.callPackage ../tools/misc/fastfetch {
+    inherit (darwin.apple_sdk_11_0.frameworks) AppKit Cocoa CoreDisplay CoreVideo CoreWLAN DisplayServices
+      Foundation IOBluetooth MediaRemote OpenCL;
```

```suggestion
    inherit (darwin.apple_sdk_11_0.frameworks)
       AppKit Cocoa CoreDisplay CoreVideo CoreWLAN DisplayServices
      Foundation IOBluetooth MediaRemote OpenCL;
```

I prefer one per line, but this should be sufficient